app/helpers/validation_helper.rb

```diff
@@ -84,11 +84,12 @@ module ValidationHelper # rubocop:todo Metrics/ModuleLength
     preferred_contact_time: %w[Morning Afternoon Evening],
     additional_planned_travel_type: %w[Domestic International],
     exposure_risk_assessment: ['High', 'Medium', 'Low', 'No Identified Risk', '', nil],
-    monitoring_reason: ['Completed Monitoring', 'Meets criteria to shorten quarantine', 'Enrolled more than 14 days after last date of exposure (system)',
-                        'Enrolled more than 10 days after last date of exposure (system)', 'Enrolled on last day of monitoring period (system)',
-                        'Completed Monitoring (system)', 'Meets Case Definition', 'Lost to follow-up during monitoring period',
-                        'Lost to follow-up (contact never established)', 'Transferred to another jurisdiction', 'Person Under Investigation (PUI)',
-                        'Case confirmed', 'Past monitoring period', 'Meets criteria to discontinue isolation', 'Deceased', 'Duplicate', 'Other', '', nil],
+    monitoring_reason: ['Completed Monitoring', 'Meets criteria to shorten quarantine', 'Does not meet criteria for monitoring',
```

Comment:
All I actually did here was add the new monitoring reason, but I had to shift things items down lines to deal with linelength for rubocop.